SDCAlertView/Source/SDCDemoViewController.m

```diff
@@ -10,7 +10,7 @@
 
 #import "SDCAlertController.h"
 #import "SDCAlertView.h"
-#import <UIView+SDCAutoLayout.h>
+#import "UIView+SDCAutoLayout.h"
```

Comment:
I tried creating a new project, installing `SDCAlertView` as a CocoaPod, but I do not get a compilation error in Xcode 7 beta 5. Can you maybe upload your project somewhere so I can see what the problem is?